README.md

```diff
@@ -105,6 +105,9 @@ pip install poetry
 # Install dependencies
 poetry install
 
+# Setup pre-commit hooks
+pre-commit install
```

Comment:
Would this be better as:

`poetry run pre-commit install`

Or should we `poetry shell` before or something else?

Reply:
Good catch! Ill take care of that in my next commit.

Reply:
To double check, on the other PR?

Also, I've really appreciated this PR (and think I'll like #19 too) - I'm not a software engineer, but am always keen to learn more! These have been tremendously educational for me, so thank you 🙌

Reply:
Updated the instruction here: 3d41505

There _might_ be a merge conflict in the other PR's `README.md` after this one goes in. If so it'll be a simple fix :)

>I'm not a software engineer, but am always keen to learn more! These have been tremendously educational for me, so thank you 🙌

Awww, brooo 😊 You are _absolutely_ a software engineer! 

I'm glad you find my code helpful! I'm happy to contribute and spread some knowledge.